playbooks/generate-csrs.yml

```diff
@@ -0,0 +1,95 @@
+---
+# Generate new certificate signing requests and keys for all cluster certificates,
+# to be delivered to a CA to sign. Keys are left in the csr staging directory
+# on each host, and csr files are copied to localhost in playbooks/csr.tgz
+
+- import_playbook: init/main.yml
+
+# Create CSRs
+- name: Create master CSRs
+  hosts: oo_masters_to_config
+  tasks:
+  - import_role:
+      name: openshift_master_certificates
+      tasks_from: csr.yml
+
+- name: Create etcd CSRs
+  hosts: oo_etcd_to_config
+  tasks:
+  - import_role:
+      name: etcd
```

Comment:
openshift_etcd_certificates ?

Reply:
that's not an actual role, the cert roles for etcd are under roles/etcd.